tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "moduleResolution": "node",
     "resolveJsonModule": false,
     "isolatedModules": false,
-    "jsx": "preserve",
+    "jsx": "react-jsx",
+    "jsxImportSource": "@emotion/react",
```

Comment:
nice! React 17 runtime! 🚀

Reply:
this was a "fun" one... a lot of heartache between babel and jest until finally stumbling on jsxImportSource which has the added boon of not needing the pragma to be added everywhere.

Reply:
yep. stumbled on this one myself a while ago. happy you were able to make it work!